packages/rx_bloc_cli/lib/src/processors/ios/podfile_processor.dart

```diff
@@ -69,14 +70,16 @@ class PodfileProcessor extends StringProcessor {
       ..insertBefore(runnerConfigs, runnerConfigsComment);
   }
 
-  void _addDeploymentTargets(StringBuffer buffer) {
-    const additionalBuildSettingsConfig =
-        'flutter_additional_ios_build_settings(target)';
-    const deploymentTargetsConfig = '''\n
+  void _addBuildConfigurations(StringBuffer buffer, GeneratorArguments args) {
```

Comment:
Do we use the `GeneratorArguments args` somewhere in the method ?

Reply:
Not anymore, it's been removed